### DISCUSSION TITLE
OFF TOPIC: Audio Sound Peak VU Meter

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#1293] [General]
Hi Team,
Does anyone know of a desktop app that simply "hears" the sound produced and coming into the PC (from ZOOM for example) and is able to show this as a peak VU meter?
Why?
Our A/V team sits a little distance from the loudspeakers at our K-Hall so what they hear differs from what the audience hear in the general auditorium seating. I thought that if we could have a peak VU meter indicating what is being input into the PC from the Mixer etc. then we could "tune" the microphone level accordingly. (Our mixer doesn't have a VU monitor feature).
We have a current workaround but I was just wondering if anyone already has something in place that is working for them... 😁

**Comment by e-Feitosa** [comment]
I think this little app will help, it can be downloaded from the Windows Store. It creates a "volume" for each PC audio output/input (see picture).

**EarTrumpet**
https://www.microsoft.com/store/productId/9NBLGGH516XP

**Printscreen**
![image](https://user-images.githubusercontent.com/90984283/213920746-a6956e7c-662a-4c75-8840-1ae36fad98ee.png)

**Reply by Chipperchap** [reply]
Thank you.
I will check it out. 😀
